fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -393,4 +393,20 @@ void FixedLagSmoother::transactionCallback(
   }
 }
 
+void FixedLagSmoother::setDiagnostics(diagnostic_updater::DiagnosticStatusWrapper& status)
+{
+  Optimizer::setDiagnostics(status);
+
+  if (status.level == diagnostic_msgs::DiagnosticStatus::OK)
```

Comment:
I had to do this because messages of `OK` level don't get merged: https://github.com/ros/diagnostics/blob/ac407b42e2e27af945d3123517c81974b2ec9986/diagnostic_updater/include/diagnostic_updater/DiagnosticStatusWrapper.h#L101-L114

Reply:
Sorry. I've been swamped the last few days.
```
/home/ec2-user/workspace/test_rosdistro_fuse_PR-108/package/fuse_optimizers/src/optimizer.cpp: In member function ‘virtual void fuse_optimizers::Optimizer::setDiagnostics(diagnostic_updater::DiagnosticStatusWrapper&)’:

[2019-09-27T19:08:56.008Z] /home/ec2-user/workspace/test_rosdistro_fuse_PR-108/package/fuse_optimizers/src/optimizer.cpp:384:19: error: ‘accumulate’ is not a member of ‘std’

[2019-09-27T19:08:56.008Z]               std::accumulate(sensor_models_.begin(), sensor_models_.end(), std::string(), print_key));

[2019-09-27T19:08:56.008Z]                    ^~~~~~~~~~

[2019-09-27T19:08:56.008Z] /home/ec2-user/workspace/test_rosdistro_fuse_PR-108/package/fuse_optimizers/src/optimizer.cpp:386:19: error: ‘accumulate’ is not a member of ‘std’

[2019-09-27T19:08:56.008Z]               std::accumulate(motion_models_.begin(), motion_models_.end(), std::string(), print_key));

[2019-09-27T19:08:56.008Z]                    ^~~~~~~~~~

[2019-09-27T19:08:56.008Z] /home/ec2-user/workspace/test_rosdistro_fuse_PR-108/package/fuse_optimizers/src/optimizer.cpp:387:33: error: ‘accumulate’ is not a member of ‘std’

[2019-09-27T19:08:56.008Z]    status.add("Publishers", std::accumulate(publishers_.begin(), publishers_.end(), std::string(), print_key));

[2019-09-27T19:08:56.008Z]                                  ^~~~~~~~~~
```
Looks like you are missing the <numeric> header.

Reply:
Thanks. Should pass now.